kroxylicious-kms-provider-aws-kms/src/main/schema/io/kroxylicious/kms/provider/aws/kms/config/AwsKmsConfig.yaml

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+$schema: http://json-schema.org/draft-04/schema#
+id: https://kroxylicious.io/schema/kms/AwsKmsConfig.yaml
+description: The AWS KMS configuration schema
+definitions:
+  PasswordProvider:
+    description: Specification of a password
+    type: object
+    properties:
+      password:
+        description: DEPRECATED. The password.
+        type: string
+      filePath:
+        description: A file path pointing to a UTF-8 encoded file containing the password
+        type: string
+    oneOf:
+      - required:
+          - password
+      - required:
+          - filePath
```

Comment:
Note nothing currently validates the plugin config against this schema, but it works because of the use of the common hand-written Java type for `PasswordProvider`. If we switched to using a generated `PasswordProvider` we'd need to also also be validating against the schema (assuming we didn't try to build some kind of subclassing support into the compiler).